drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -113,6 +113,12 @@ def self.configuration
       can :view_any_confidential_client_files
     end
 
+    [:first_name, :middle_name, :last_name, :name_suffix].each do |attr|
+      define_method(attr) do
+        object.send(attr)&.titleize
```

Comment:
Hmm, I think you should preserve they casing as entered.  Sometimes people have multiple sequential capitals in their name, or potentially have a reason to want all lower case, or a name like de Angelo or D'Angelo....  Like a phone number, I guess I tend to trust the entry over anything else.